server/bin/unittests

```diff
@@ -334,6 +334,11 @@ function _setup_state {
         exit $res
     fi
 
+    # Ensure proper expected umask.
+    find ${_testopt} ${_testopt_sat} \
+            \( -perm -u=rx -exec chmod 775 {} \+ \) -o \
+            \( -perm -u=r ! -perm -u=rx -exec chmod 664 {} \+ \)
```

Comment:
FWIW, only one branch of the `-o` will execute, so you don't need the `! -perm` in the second branch:
```
    find ${_testopt} ${_testopt_sat} \
            \( -perm -u=rx -exec chmod 775 {} \+ \) -o \
            \( -perm -u=r -exec chmod 664 {} \+ \)
```